makeitdark.py

```diff
@@ -1,7 +1,16 @@
-from sys import platform
+from sys import platform, argv
 import os
+if len(argv) != 1:
+    if argv[1] == "makeitlight":
+        undo_mode = True
+    else:
+        print("run 'python ./makeitdark.py makeitlight' to undo the changes")
+        exit()
+else:
+    undo_mode = False
 
-injectable = "document.addEventListener(\"DOMContentLoaded\", function() {  \n\
+injectable = "/* BEGIN makeitdark */ \n\
```

Comment:
Ideally we can opt to declare the perimeter tags as constants here.
Accepting this though, it's a much needed change.